Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -0,0 +1,272 @@
+                    ChangeWizard(targetWizard);
+                }
+                else
+                {
+                    Debug.Log("Error");
+                }
+            });
+            _container.Q<Button>("UserProfilePageAccountNewProfileCancelButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                ClearCredentials();
+                SetupBootMenu();
+            });
+            _container.Q<Button>("UserProfilePageBootstrapStartButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                BootstrapSettings.RefreshBucketName();
+                OpenS3Popup(BootstrapSettings.BucketName);
+            });
+            _container.Q<Button>("UserProfilePageBootstrapAnotherBucketButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                OpenS3Popup(_stateManager.BucketName);
+            });
+            _container.Q<Button>("AccessKeyToggleReveal").RegisterCallback<ClickEvent>(_ =>
+            {
+                var accessToggle = _container.Q<TextField>("AccessKeyField");
+                ToggleHiddenText(accessToggle);
+            });
+            _container.Q<Button>("SecretKeyToggleReveal").RegisterCallback<ClickEvent>(_ =>
+            {
+                var secretToggle = _container.Q<TextField>("SecretKeyField");
+                ToggleHiddenText(secretToggle);
+            });
+            _container.Q<Button>("AddProfile").RegisterCallback<ClickEvent>(_ =>
+            {
+                var targetWizard = _container.Q<VisualElement>("AddNewProfile");
+                ChangeWizard(targetWizard);
+            });
+        }
+
+        private void OpenLink(string url)
+        {
+            Application.OpenURL(url);
+        }
+
+        private void SetupTab()
+        {
+            SetupBootMenu();
+        }
+
+        private void SetupConfigSettings()
+        {
+            var selectedProfile = _stateManager.CoreApi.GetSetting(SettingsKeys.CurrentProfileName);
+            _stateManager.SelectedProfile = selectedProfile.Success ? selectedProfile.Value : _stateManager.AllProfiles.First();
+        }
+
+        private void SetupBootMenu()
+        {
+            VisualElement targetWizard;
+            // var tab2WarningBox = _container.Q<VisualElement>(null, "Tab2Warning");
+            var cardsMenu =  _container.Q<VisualElement>("Cards");
+            var newProfileMenu = _container.Q<VisualElement>("AddNewProfile");
+            var bootStrapMenu = _container.Q<VisualElement>("BootstrapMenu");
+            var completedMenu = _container.Q<VisualElement>("CompletedProfile");
+            cardsMenu.AddToClassList(hiddenClassName);
+            newProfileMenu.AddToClassList(hiddenClassName);
+            bootStrapMenu.AddToClassList(hiddenClassName);
+            completedMenu.AddToClassList(hiddenClassName);
+            AccountDetailTextFields = newProfileMenu.Query<TextField>().ToList();
```

Comment:
Why are we assigning this field in this method?

Reply:
this method has been refactored now